engine/framework/base_jobmaster.go

```diff
@@ -275,13 +273,10 @@ func (d *DefaultBaseJobMaster) Workload() model.RescUnit {
 	return d.worker.Workload()
 }
 
-// ID delegates the ID of inner worker
-func (d *DefaultBaseJobMaster) ID() runtime.RunnableID {
-	return d.worker.ID()
-}
```

Comment:
This function signature should not be changed.

Reply:
Yes!! You are right. Fix it.
BTW, I think it's strange for the `ID()` of `Worker` to return a runtime-related type `runtime.RunnableID`, but not a `WorkerID`.